src/actions/web3.js

```diff
@@ -92,15 +92,23 @@ export const getLogs = address => async (dispatch, getState) => {
     return null;
   }
   const tokenContract = icoConfig.tokenContract ? getTokenSmartContract(web3, address) : null;
+  const contracts = {};
```

Comment:
you can write it using object literal:
const contracts = {
[address]=icoContract,
[icoConfig.tokenContract]= tokenContract,
};

Reply:
OMG what a syntax with those square brackets, I somehow see where it is coming from... fixed